src/components/Reports/ReportsNew.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import Reports from './ReportsNew';
+
+describe('Reports New page', () => {
+  afterEach(cleanup);
+
+  it('render page', () => {
+    // Arrange
+    // Act
+    const { getByText } = render(
+      <DopplerIntlProvider>
+        <Reports />
+      </DopplerIntlProvider>,
+    );
+
+    // Assert
+    expect(getByText('reports_title'));
```

Comment:
This means that it should find exactly the text 'reports_title' in the document, doesn't it? In this case the id="reports_title"

Reply:
No, it says that the text `reports_title` will be visible in the rendered output for the user, not in the source code.

### Explanation

I do not totally agree, but the philosophy behind React Testing Library [says "The queries help you to find elements in the same way that users will find them"](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change).

So, in general we find elements by text.

We have a problem with the localization and language, for that reason, we have a special _Intl Provider_:

```
 import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
```

It returns the keys in place of the values. 

So, if you test this page (even when it will be modified) you probably have this text:

![image](https://user-images.githubusercontent.com/1157864/80402911-0f837700-8895-11ea-9c7a-038fa0e1b522.png)

### Regrets

Oh,  but now I am understanding that `reports_title` text will be in the header, so our test is not so robust :(

Reply:
Ok, maybe "document" wasn't the best word, but the idea is to find the text in the rendered output. 
I didn't uderstand what you mean about "reports_title will be in the header" (I didn't find it in header component 😕)